.coveragerc

```diff
@@ -38,6 +38,7 @@ omit =
     homeassistant/components/apple_tv/*
     homeassistant/components/aqualogic/*
     homeassistant/components/aquostv/media_player.py
+    homeassistant/components/arcam_fmj/*
```

Comment:
Make sure you don't exclude the tests for your config flow...

Reply:
As noted below, there is not real config flow. Only import. Since device registry is non working anyway for this integration (can't get unique id) i could just drop it and move back to old way?

Reply:
We should still test the config flow. That module needs 100% coverage.